test/manipulation.test.js

```diff
@@ -967,6 +967,13 @@ describe('manipulation', function() {
         });
       });
 
+      function changePostIdInBeforeSaveHook(operationHook, ctxField) {
+        Post.observe(operationHook, function(ctx, next) {
+          ctx[ctxField].id = 99;
```

Comment:
Can we infer `cxField` ourselves? The hooks always provide only one of `ctx.data` or `ctx.instance` IIRC.

```js
(ctx.data || ctx.instance).id = 99;
next();
```